### PR TITLE
Pan 18562: add fpe context to result

### PR DESCRIPTION
**Pull request by ajpangea**

### CODE REVIEW

File: packages/pangea-sdk/pangea/services/redact.py [hunk 1]
```diff
@@ -260,6 +262,8 @@ def redact(
             rulesets (list[str], optional): An array of redact rulesets short names
             return_result(bool, optional): Setting this value to false will omit the redacted result only returning count
             redaction_method_overrides: A set of redaction method overrides for any enabled rule. These methods override the config declared methods
+            llm_request: Boolean flag to enable FPE redaction for LLM requests
```
Comment (pangea-andrest) [inline packages/pangea-sdk/pangea/services/redact.py:265]:
```suggestion
            llm_request (bool, optional): Boolean flag to enable FPE redaction for LLM requests
```

File: packages/pangea-sdk/pangea/services/redact.py [hunk 2]
```diff
@@ -317,6 +321,8 @@ def redact_structured(
...
```
Comment (pangea-andrest) [inline packages/pangea-sdk/pangea/services/redact.py:324]:
```suggestion
            llm_request (bool, optional): Boolean flag to enable FPE redaction for LLM requests
```

**Review by pangea-andrest — APPROVED** [review]

**Comment by kenany** [comment]
Rebased onto latest `main`, squashed, added missing changes to the asyncio submodule, fixed up the changelog a little, and added a release commit for v5.2.1.

**Review by kenany — APPROVED** [review]

File: packages/pangea-sdk/pangea/services/redact.py (hunk 1 above)
Comment (kenany) [reply packages/pangea-sdk/pangea/services/redact.py:265]: Docs generator can infer this from the types for a while now, so this sort of thing is mostly useless AFAIK.

**Pull request merged**